hyundai_kia_connect_api/KiaUvoApiCA.py

```diff
@@ -118,6 +119,15 @@ def get_cached_vehicle_status(self, token: Token):
 
         vehicle_status = {}
         vehicle_status["vehicleStatus"] = response
+        
+        #Coverts temp to usable number.  Currently only support celsius. Future to do is check unit in case the care itself is set to F. 
+        
+        value = vehicle_status["vehicleStatus"]["airTemp"]["value"]
```

Comment:
this looks like a perfect function into util.py eg;
```
get_hex_temp_into_index(value):
    value = value.replace("H", "")
    value = value.replace("C", "")
    value = "0x" + value
    value = int(value,16)
```

Reply:
Done.